DockerfileMakePackage

```diff
@@ -78,6 +82,7 @@ RUN find /ovms_release/lib/ -iname '*.so*' -exec patchelf --debug --set-rpath '$
 WORKDIR /ovms
 RUN cp -v /ovms/release_files/LICENSE /ovms_release/
 RUN cp -rv /ovms/release_files/thirdparty-licenses /ovms_release/
+RUN mkdir -vp /ovms_release/include && cp /ovms/src/pocapi.h /ovms_release/include
```

Comment:
Do we have source files in release image? /ovms/src/

Reply:
We use build image: BUILD_IMAGE=$(OVMS_CPP_DOCKER_IMAGE)-build